foxtrot-server/src/main/resources/console/index.htm

```diff
@@ -311,9 +297,7 @@ <h4 class="modal-title" id="saveConsoleModalLabel">Save Console</h4>
 
               <div class="modal-footer">
                   <div class="checkbox" style="float: left;">
-                      <label>
-                          <input type="checkbox" id="pinConsoleToApp">Pin console to current app
-                      </label>
+                      <label> <input type="checkbox" checked id="pinConsoleToApp">Pin console to current app</label>
```

Comment:
By default it should be checked since mostly console creators would want to pin console to app